ansible/inventory/group_vars/operator-pipeline-prod.yml

```diff
@@ -1,6 +1,7 @@
 env: prod
 ocp_host: https://api.pipelines-prod.ijdb.p1.openshiftapps.com:6443
 branch: main
+---
```

Comment:
Shouldn't this be at the top of this file?

Reply:
Fixed